pipelines/pipeline_katello_32.yml

```diff
@@ -32,38 +29,36 @@
     - disable_firewall
     - haveged
     - update_os_packages
-    - foreman_installer
 
 - hosts: pipeline-katello-3.2-centos7
   become: yes
   vars:
     foreman_installer_scenario: katello
-    foreman_installer_options_internal_use_only:
+    foreman_installer_options:
       - "--disable-system-checks"
-      - "--foreman-admin-password {{ foreman_installer_admin_password }}"
-    foreman_installer_additional_packages:
-      - katello
+    installer_scenarios_answers:
+      foreman:
+        admin_password: changeme
```

Comment:
This makes the role idempotent, if you modify `foreman_installer_options` you have to run the installer to make sure its applied (everytime), wheras this method allows you to manage the answer file and run the installer as a handler (only if something changes). https://github.com/sean797/ansible-role-foreman_installer/blob/master/templates/scenario-answers.yaml.j2